src/Datadog.Trace/TraceIdExtensions.cs

```diff
@@ -0,0 +1,40 @@
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// Extensions for <see cref="TraceId"/> class.
+    /// </summary>
+    public static class TraceIdExtensions
```

Comment:
Can we add tests for these methods? I couldn't find any in this PR.

Reply:
Good catch, thanks!